src/test/java/com/wikia/webdriver/testcases/infoboxbuilder/InfoboxBuilderTests.java

```diff
@@ -108,41 +125,51 @@ public void deletingAddedComponents() {
     int imageComponents = builderPage.countImages();
     int headerComponents = builderPage.countHeaders();
 
-    builderPage.addImageComponent().addTitleComponent().addRowComponent().addHeaderComponent();
+    builderSidebar.addImageComponent().addTitleComponent().addRowComponent().addHeaderComponent();
 
     /* deleting last (newly added) components */
-    Assertion.assertEquals(
-        rowComponents,
-        builderPage.deleteRowUsingButton(builderPage.countRows() - 1).countRows()
-    );
-    Assertion.assertEquals(
-        titleComponents,
-        builderPage.deleteTitleUsingButton(builderPage.countTitles() - 1).countTitles()
-    );
-    Assertion.assertEquals(
-        imageComponents,
-        builderPage.deleteImageUsingButton(builderPage.countImages() - 1).countImages()
-    );
-    Assertion.assertEquals(
-        headerComponents,
-        builderPage.deleteHeaderUsingButton(builderPage.countHeaders() - 1).countHeaders()
-    );
+    builderPage.selectRowWithIndex(builderPage.countRows() - 1);
+    builderSidebar.clickDeleteButton();
+    Assertion.assertEquals(rowComponents, builderPage.countRows());
+
+    builderPage.selectTitleWithIndex(builderPage.countTitles() - 1);
+    builderSidebar.clickDeleteButton();
+    Assertion.assertEquals(titleComponents, builderPage.countTitles());
+
+    builderPage.selectImageWithIndex(builderPage.countImages() - 1);
+    builderSidebar.clickDeleteButton();
+    Assertion.assertEquals(imageComponents, builderPage.countImages());
+
+    builderPage.selectHeaderWithIndex(builderPage.countImages() - 1);
+    builderSidebar.clickDeleteButton();
+    Assertion.assertEquals(headerComponents, builderPage.countHeaders());
   }
 
   @Execute(asUser = User.USER)
   public void customizingComponents() {
     TemplatePage template = new TemplatePage();
+    Sidebar builderSidebar = new Sidebar();
+    String labelText = "AutomatedTest";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "AutomatedTest" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)